README.md

```diff
@@ -54,17 +71,36 @@ Curation of papers is generally based on the recent publication in top AI confen
 - **Bayesian-optimized Bidirectional LSTM Regression Model for Non-intrusive Load Monitoring** [[link]](https://ieeexplore.ieee.org/document/8683110)
   - Maria Kaselimi, Nikolaos Doulamis, Anastasios Doulamis, Athanasios Voulodimos, Eftychios Protopapadakis. *ICASSP'19*
 
-### Others
+#### Others
 - **Gemello: Creating a Detailed Energy Breakdown from just the Monthly Electricity Bill** [[pdf]](https://www.kdd.org/kdd2016/papers/files/adp1036-batraA.pdf)
   - Nipun Batra, Amarjeet Singh, Kamin Whitehouse. *KDD'16*
 - **Disambiguating Energy Disaggregation: A Collective Probabilistic Approach** [[pdf]](https://www.ijcai.org/proceedings/2017/0398.pdf)
   - Sabina Tomkins, Jay Pujara, Lise Getoor. *IJCAI'17*
+- **On the Applicability of Correlation Filters for Appliance Detection in Smart Meter Readings** [[pdf]](http://mobile.nes.aau.at/publications/klemenjak-2017-Correlation_Filters.pdf)
```

Comment:
Could you explain why this paper should be included in this list? Up to now, I mainly listed the papers published in the AI, data mining top conferences. If you want to include your own paper, I hope you explain the impactfulness of this paper.

Reply:
Just a suggestion. We implemented a simple correlation filter approach for NILM. The main idea was to provide a simple reference implementation for more sophisticated NILM algorithms. Feel free to ignore this paper if it doesn't meet your expectations.